taxi/tests/test_views.py

```diff
@@ -68,3 +67,25 @@ def test_queryset_with_search_criteria(self):
             list(response_empty_criteria.context.get("manufacturer_list")),
             list(non_filtered_manufacturer_list),
         )
+
+
+class ToggleAssignToCarTest(TestCase):
+    def setUp(self) -> None:
+        self.user = get_user_model().objects.create_user(
+            username="TestUser", password="123456789"
+        )
+        self.client.force_login(self.user)
+        self.manufacturer = Manufacturer.objects.create(
+            name="VolksWagen", country="Germany"
+        )
+        self.car = Car.objects.create(
+            model="Golf", manufacturer=self.manufacturer
+        )
+
+    def test_toggle_assign_to_car_driver_in_car_drivers(self):
+        self.client.get(reverse("taxi:toggle-car-assign", args=([1])))
+        self.assertIn(self.user, self.car.drivers.all())
+
+    def test_toggle_assign_to_car_driver_not_in_car_drivers(self):
+        self.client.get(reverse("taxi:toggle-car-assign", args=([1])))
+        self.assertIn(self.user, self.car.drivers.all())
```

Comment:
test for checking driver not in car, but u are using assertIn ? both tests are similar

Reply:
> test for checking driver not in car, but u are using assertIn ? both tests are similar

Yep, my mistake, Thank's